truelearn/utils/visualisations/_pie_rose_plotter.py

```diff
@@ -234,10 +284,9 @@ def plot(
             )
```

Comment:
for 283 expected type dict got str
not sure if this is a correct warning

Reply:
Maybe not? Based on the generated chart, it seems that it's correct.